pkg/infrastructure/services/dashboards.go

```diff
@@ -136,13 +143,11 @@ func deployGrafanaDashboards(dashboards []GrafanaDashboard, cli *client.Client,
 				Name: dashboard.Name,
 			},
 		}
-		isCreated, err := kubernetes.ResourceC(cli).CreateIfNotExistsForOwner(dashboardDefinition, kogitoService, scheme)
-		if err != nil {
+		if err := kubernetes.ResourceC(cli).CreateIfNotExistsForOwner(dashboardDefinition, kogitoService, scheme); err != nil {
 			log.Warnf("Error occurs while creating dashboard %s, not going to reconcile the resource: %v", dashboard.Name, err)
+			return err
 		}
-		if isCreated {
-			log.Infof("Successfully created grafana dashboard %s", dashboard.Name)
-		}
+		log.Infof("Successfully created grafana dashboard %s", dashboard.Name)
```

Comment:
The reason why this log was executed only in case isCreated is true was that `deployGrafanaDashboards` is called in every reconciliation cycle. So every reconciliation contained `Successfully created grafana dashboard` log entry even when no dasboard was created in that cycle.

Reply:
That's true. I'll remove this log line. WDYT?

Reply:
ok for me